### PR TITLE
chore: move tests to origin pull request, update and clean also

### PR DESCRIPTION
**Pull request by alejandroEsc**
Here we clean a simplify the license test to be part of the origin tests.

### CODE REVIEW

File: .github/workflows/pull_requests_connectors.yaml [hunk 1]
```diff
@@ -24,7 +24,12 @@ on:
       - .github/ct-connectors.yaml
       - .github/kind.yaml
       - .github/workflows/pull_requests_connectors.yaml
+      - charts/**
       - 'charts/connectors/**'
+      - '!charts/console/**'
```
[the remark this answers is not in the document]
Comment (alejandroEsc) [reply .github/workflows/pull_requests_connectors.yaml:29]: i added 'charts/**', so yes.

File: .github/workflows/pull_requests_from_origin.yaml [hunk 2]
```diff
@@ -25,6 +25,10 @@ on:
       - .github/kind.yaml
       - .github/workflows/pull_requests_from_origin.yaml
       - charts/**
+      - 'charts/console/**'
```
[the remark this answers is not in the document]
Comment (alejandroEsc) [reply .github/workflows/pull_requests_from_origin.yaml:28]: sorry I was not aware this was the connectors pipeline?

File: .github/workflows/pull_requests_redpanda.yaml [hunk 3]
```diff
@@ -46,7 +51,7 @@ jobs:
       - name: Set up Helm
         uses: azure/setup-helm@v3
         with:
-          version: v3.11.1
+          version: v3.12.2
```
Comment (RafalKorepta) [inline .github/workflows/pull_requests_redpanda.yaml:54]: Good change. Can this be applied to all workflows?

**Comment by alejandroEsc** [comment]
I am going to merge this PR in. But there still remains some flakyness in other tests that are NOT addressed here.

**Comment by RafalKorepta** [comment]
Lint test's are false negative. We don't need to change chart version if there is no changes to the chart itself.

**Pull request merged**